arelle/ModelDtsObject.py

```diff
@@ -1997,20 +1997,10 @@ def isCovered(self):
             return self._isCovered
 
     @property
-    def axisDisposition(self):
```

Comment:
arbitrary line for 🧵 discussion:

> I have created #316 to fix some python errors. There is still a lot of exception in the table linkbase conformance suite, but it seems more linked to the tests itself, so it should be corrected when making the conformance pass.
> 
> @aaroncameron-wk For the table linkbase conformance suite, the 2014-03-18 version is used (https://github.com/Arelle/Arelle/blob/master/tests/integration_tests/validation/XBRL/test_xbrl_table_linkbase_conformance_suite.py#L6), but there is a newer one from 2015-08-12 (https://specifications.xbrl.org/release-history-table-linkbase-1.0-conformance-suite.html) do you know why it's not the latest used in the tests?

There doesn't seem to have been a good reason for this. Aaron created #323. I'll get this prioritized on our side